content/docs/user-guide/experiment-management/checkpoints.md

```diff
@@ -0,0 +1,454 @@
+generate a `plots.html` file that you can open in a browser and it will display
+plots for you similar to the ones below.
+
+![Plots generated from running experiments on MNIST dataset using DVC](/img/checkpoints_plots.png)
+_The results here won't show anything interesting because the model increases in
+accuracy so fast._
+
+## Resetting checkpoints
+
+Usually when you start training a model, you won't begin with accuracy this
+high. There might be a time when you want to remove all of the existing
+checkpoints to start the training from scratch. You can reset your checkpoints
+with the following command:
+
+```bash
+dvc exp run --reset
+```
+
+This resets all of the existing checkpoints and re-runs the code to generate a
+new set of checkpoints under a new experiment branch.
+
+```dvc
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━┳━━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━┳━━━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created  ┃ step ┃    loss ┃    acc ┃ seed   ┃ lr     ┃ weight_decay ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━╇━━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━╇━━━━━━━━━━━━━━┩
+│ workspace             │ -        │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ main                  │ 04:49 PM │    - │       - │      - │ 473987 │ 0.001  │ 0            │
+│ │ ╓ exp-a75bb         │ 05:18 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b8a4ceb           │ 05:18 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 7b652b1           │ 05:17 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ cf0168f           │ 05:17 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ 8c7f40f           │ 05:17 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+│ │ ╓ exp-7fafd         │ 05:10 PM │    5 │ 0.36601 │ 0.8943 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 9ac9e6a           │ 05:09 PM │    4 │ 0.41666 │ 0.8777 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ b869efe           │ 05:09 PM │    3 │ 0.50835 │ 0.8538 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 422395e           │ 05:09 PM │    2 │ 0.72421 │ 0.8284 │ 473987 │ 0.0001 │ 0            │
+│ │ ╟ 44e44a9           │ 05:09 PM │    1 │  1.2537 │ 0.7738 │ 473987 │ 0.0001 │ 0            │
+│ ├─╨ c6e11b4           │ 05:09 PM │    0 │  1.9428 │ 0.5715 │ 473987 │ 0.0001 │ 0            │
+```
+
+We can also remove all of the experiments we don't promote to our Git workspace
+with the following command:
+
+```bash
+dvc exp gc --workspace --force
+```
+
+## Adding checkpoints to Git
+
+When you terminate training, you'll see a few commands in the terminal that will
+allow you to add these changes to Git.
+
+```dvc
+To track the changes with git, run:
+
+        git add dvclive.json dvc.yaml .gitignore train.py dvc.lock
+
+Reproduced experiment(s): exp-263da
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
```

Comment:
If you want to keep all of the checkpoints as commits in your Git history, you can use `dvc exp branch` to do that. I'm torn on whether it's worthwhile to explain that here. cc @dmpetrov @jorgeorpinel

Reply:
I wouldn't mention `exp branch` out of context here. Actually I'd skip the whole code block above, as the `git` command is repeated below.